PID-Analyzer.py

```diff
@@ -125,7 +125,11 @@ def winmaker(self, flen):
 
     def wiener_deconvolution(self, input, output, smooth):      # input/output are twosimensional
         pad = len(input[0]) - 2*(len(input[0])%1024)            # padding to potence of 2, increases transform speed
```

Comment:
Good thing you saw this. It should have been impossible to get nagative padding values, so the calculation is wrong. The correct calculation should be
 `pad = len(input[0]) + (1024-(len(input[0])%1024))`
This will always be >0 and a multiple of 1024. 
It's useful because for some odd numbers the numpy fft is slower (caused by some inefficient internal reshaping/padding).

Reply:
Ah hah, that makes more sense. Updated.